src/mdlComponentHandler.js

```diff
@@ -86,6 +86,15 @@ var componentHandler = (function() {
    * the element to.
    */
   function upgradeElementInternal(element, jsClass) {
+    if (jsClass === undefined) {
+      for (var i = 0; i < registeredComponents_.length; i++) {
```

Comment:
I'd say we leave it as-is for now. and if we decide to refactor `forEach` in later we knock it all out at once. Looks like IE9 even has support for that, so it is plausible.